protocols/GatkGenotypeGvcf.sh

```diff
@@ -41,7 +42,7 @@ tmpProjectBatchCombinedVariantCalls=${MC_tmpFile}
 
 #Load modules
 ${stage} "${gatkVersion}"
-
+${stage} "${HTSLibVersion}"
 #Check modules
 ${checkStage}
```

Comment:
here you are using ${checkStage}, whereas you replaced it with module list in one of the protocols above